WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -581,7 +585,17 @@ public bool IsCustomFee
 		public bool IsCustomChangeAddress
 		{
 			get => _isCustomChangeAddress;
-			private set => this.RaiseAndSetIfChanged(ref _isCustomChangeAddress, value);
+			private set
+			{
+				if (IsMax)
```

Comment:
Do not mix concepts. One property, one function:

- IsCustomChangeAddress should be a get only property.
- Create a WhenAnyValue Subscribe on Global.UiConfig.IsCustomChangeAddress and IsMax
- In the Susbscribe trigger the reevaluation of IsCustomChangeAddress with the RaisePropertyChanged method

Reply:
I have done some changes (https://github.com/zkSNACKs/WalletWasabi/pull/3544/commits/b8ed87fe00b75e03337cad1af8241d1b001f16cc).
Can you take a look if the logic is correct?

I didn't rename the property yet.